tools/project_manager/page_compare.php

```diff
@@ -43,7 +43,7 @@ function render()
         output_header("$title: $sub_title", NO_STATSBAR);
 
         echo "<h1>$title</h1>\n";
-        echo "<h2>$sub_title</h2>\n";
+        echo "<h2>".html_safe($sub_title)."</h2>\n";
```

Comment:
Please put spaces around concatenation (here and elsewhere you've added them), eg:
```
echo "<h2>" . html_safe($sub_title) . "</h2>\n";
```

Reply:
Yeah I mimicked what I saw somewhere else. I'll update all my changed string concatenations and verify runtime dependencies are met.

Reply:
Updated